doc/admin-guide/plugins/xdebug.en.rst

```diff
@@ -90,6 +90,10 @@ X-Cache
     skipped     The cache lookup was skipped.
     ==========  ===========
 
+    If a request goes through multiple proxies, each one prepends its X-Cache header content
+    at the beginning of the existing X-Cache header. As a result, the order is reversed from
+    the Via: header.
```

Comment:
Nice. I *wonder*, since the behavior of the Via header is pretty darn well known (and defined), should we not adhere to this for X-Cache too ? What does Akamai do ?

Reply:
At least for us, Akamai does not expose multiple layers for their equivalent, X-cache and X-cache-remote -- both are single layer only.